modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -785,55 +795,228 @@ class NDB_Form_Statistics extends NDB_Form
+            	            s.SubprojectID,
+            	            c.CenterID,
+            	            s.Visit_label)";
+            }
+            $PRO_params['no_scans']        = $no_scans;
+            $PRO_params['partial_scans']   = $partial_scans;
+            $PRO_params['complete_scans']  = $complete_scans;
+            $PRO_params['suproject_query'] = $suproject_query;
+            $PRO_params['Param_Project']   = $Param_Project;
+            $PRO_params['unspec_scans']    = $unspec_scans;
+
+            $Pre_Result_Two   = "CREATE TEMPORARY TABLE IF NOT EXISTS temp_comp AS (
+                    SELECT
+                        t.SubprojectID,
+                        t.CenterID,
+                        t.VLabel,
+                        \"Complete\" AS Subcat,
+                        t.total_comp AS val
+                    FROM temp_totals t)";
+            $Pre_Result_Three = "CREATE TEMPORARY TABLE IF NOT EXISTS temp_part AS (
+                    SELECT
+                        t.SubprojectID,
+                        t.CenterID,
+                        t.VLabel,
+                        \"Partial Run\" AS Subcat,
+                        t.total_part AS val
+                    FROM temp_totals t)";
+            $Pre_Result_Four  = "CREATE TEMPORARY TABLE IF NOT EXISTS temp_no AS (
+                    SELECT
+                        t.SubprojectID,
+                        t.CenterID,
+                        t.VLabel,
+                        \"No Scan\" AS Subcat,
+                        t.total_no AS val
+                    FROM temp_totals t)";
+            $Pre_Result_Five  = "CREATE TEMPORARY TABLE IF NOT EXISTS temp_unsp AS (
+                    SELECT
+                        t.SubprojectID,
+                        t.CenterID,
+                        t.VLabel,
+                        \"Unspecified\" AS Subcat,
+                        t.total_unsp AS val
+                    FROM temp_totals t)";
+            $Result_query     = "SELECT * FROM temp_comp UNION SELECT * FROM";
+            $Result_query    .= " temp_part UNION SELECT * FROM temp_no";
+            $Result_query    .= " UNION SELECT * FROM temp_unsp";
+            $DB->execute(
+                $DB->prepare($Pre_Result_One),
+                $PRO_params,
+                array('nofetch' => true)
+            );
+            $DB->execute(
+                $DB->prepare($Pre_Result_Two),
+                array(),
+                array('nofetch' => true)
+            );
+            $DB->execute(
+                $DB->prepare($Pre_Result_Three),
+                array(),
+                array('nofetch' => true)
+            );
+            $DB->execute(
+                $DB->prepare($Pre_Result_Four),
+                array(),
+                array('nofetch' => true)
+            );
+            $DB->execute(
+                $DB->prepare($Pre_Result_Five),
+                array(),
+                array('nofetch; => true')
```

Comment:
I think this line is a typo (it's not the same syntax as all the ones above it..), but since I'm not sure why it's there I can't say what it'll affect.

Reply:
it is a typo good catch